pennylane/templates/transforms.py

```diff
@@ -0,0 +1,21 @@
+from functools import wraps
+from pennylane.tape.tapes import QuantumTape
+
+
+def adjoint(fn):
+    """Create a function that applies the adjoint of the provided operation or template.
+
+    **Example**
+
+     ``adjoint(fn)(args)`` will apply all of the operations executed during ``fn(args)``,
+     but in reverse and with each operation adjointed.

```

Comment:
Can you add a minimal code example here? We have a goal to have full working examples for every docstring in PL

Reply:
Added.